Hello Microverse Project Pull Request

Simiran987 opened the pull request.

franclobo reviewed (changes requested):
Hi @Simiran987, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

- ## **This is a re-review and I couldn't see the last reviews.**

- [x] Please 🙏🏽 make all the changes in the corresponding branch not directly in the main branch for a correct GitHub flow.
- [ ] Kindly add the **acknoledgment** and **license** sections these are needed in the README file for a professional README file.
- [ ] Kindly add the LICENSE file to your repository and make sure that the link in the README file works as expected.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
# _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Simiran987 commented:
@franclobo With due respect, I wanted to change the HTML and CSS files in my first pull request and accidentally deleted the whole branch and that's why I didn't have any other option than to create an entirely new branch. Also, I have incorporated the suggested changes and added the acknowledgement and license sections into the README.file. Sorry for the inconvenience!!

sinansevgi reviewed (commented):
Hello @Simiran987  👋,

_This is an additional review as a result of the second opinion request._

It took Thomas Edison 1000 tries to invent the electric bulb. It also means that it took him 999 mistakes and one successful attempt. Making errors is a [crucial part of the learning process](https://pubmed.ncbi.nlm.nih.gov/23815457/). 
I also want to point out that you are not alone here and you can always ask other students or [us](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-) for help. 

I noticed that you already worked on some of the requested changes by @franclobo, but however, there are some parts that still need some attention.
- I noticed that you add a license to the repository (main branch) but the license link on your readme doesn't work. It looks like the link leads us to a file named "LICENSE", instead of the MIT.md file. If you are having issues with the markdown syntax that we are using for editing readme files, you can check [this simple guide](https://www.markdownguide.org/basic-syntax/#links) about markdown syntax.
- I couldn't find the `index.html` file or CSS file on both of your branches, looks like you removed them. Please be sure that you have the `index.html` file on your repository
. Since these files are main project files we can't review your project properly without these files. Please be sure that you have these files in your repository (Looks like you have removed all files from the branch)

Please make these changes and request another review.


Keep rocking 🚀!

HAMMAS-SALEEM commented on index.html:
```diff
@@ -0,0 +1,18 @@
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
+    <img src="https://media.giphy.com/media/JmBXdjfIblJDi/giphy.gif" alt="A Cute Giraffe saying Hi">
+    <h2>About Me</h2>
+    <hr>
+      <ol>
+        <li><strong>Name:</strong> Simiran Mohapatra</li>
+        <li><strong>Country:</strong> India</li>
+        <li><strong>Education:</strong> B.Tech (Electrical Engineering</li>
+        <li><strong>Hobbies:</strong> Dancing, Reading, listening to music, writing poems and sketching.</li>
+        <li><strong>Hogwarts House:</strong> Ravenclaw</li>
+      </ol>
+      <pre class="center"><em>''I always wanted to learn coding, work as a techie and become a tech entrepreneur.<br>
+        So, here I am to learn how to be a good web and software developer.<br>
+        I am glad to be a part of the September Cohort & I aspire to give my best!!''</em></pre>
+</body>
+</html>
```
**[Optional]**
- [ ] You are not following the syntax of HTML. You can see that [here](https://www.w3schools.com/html/html_intro.asp) :blush:

HAMMAS-SALEEM reviewed (approved):
# Status: Approved :green_circle:
![success](https://camo.githubusercontent.com/6ada59f6189497c1c2cae6d06f8154b7307a58643f232340e288140ed469eb28/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966)

## Hey @Simiran987 :wave:

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Following the GitHub Flow
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Descriptive PR Title

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
